Formula/kenv.rb

```diff
@@ -0,0 +1,54 @@
+class Kenv < Formula
+  desc "Environment file injector for Kubernetes resources"
+  homepage "https://github.com/thisendout/kenv"
+  url "https://github.com/thisendout/kenv/archive/v0.3.1.tar.gz"
+  sha256 "c245583aee13cb0e1b6ec44e7d16fe83455fbca2657b73f49f1d7328b8594204"
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    bin_path = buildpath/"src/github.com/thisendout/kenv"
+    bin_path.install Dir["*"]
+    cd bin_path do
+      system "go", "build", "-o", bin/"kenv", "."
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.